common/src/dodeca.rs

```diff
@@ -1,9 +1,12 @@
 //! Tools for processing the geometry of a right dodecahedron
-
-use lazy_static::lazy_static;
 use serde::{Deserialize, Serialize};
 
-use crate::math;
+use crate::dodeca::data::*;
+
+pub const VERTEX_COUNT: usize = 20;
+pub const SIDE_COUNT: usize = 12;
+pub const BOUNDING_SPHERE_RADIUS_F64: f64 = 1.2264568712514068;
+pub const BOUNDING_SPHERE_RADIUS: f32 = BOUNDING_SPHERE_RADIUS_F64 as f32;
```

Comment:
Try to avoid moving code around if you don't need to. These were originally underneath `impl Vertex`. Moving them forces code reviewers to verify that no unintended changes were made to them, and makes `git blame` output less useful.